test/locales/README.md

```diff
@@ -0,0 +1,8 @@
+# Locale files for Playwright tests
+
+Normally when you build Openverse, you download the updated information about locales and translation files to `src/locales` folder. To prevent too many requests during CI, we save the locale files that are necessary to run the app during Playwright tests.
+
+1. `scripts/valid-locales.json` contains a list of valid locales with properties including their translation status (the percentage of strings that are translated).
+2. `es.json` is the `es` locale file used for search navigation tests (`../playwright/e2e/search-navigation.spec.ts`).
+3. `ru.json` is the `ru` locale file used for testing the translation banner because the current translated percentage is ~40%, which is below the banner threshold of 90%.
+4. `ar.json` is the machine-translated `ar` locale file used for RTL testing.
```

Comment:
For completion's sake, should we also pull in a partially machine translated Urdu that is below the banner threshold so we have an RTL one as well?

Reply:
Actually, we can simulate untranslated Urdu by adding it to `valid-locales` with `translated` set to 0, and adding an empty `ur.json` file (or setting `translated` to a low percentage and adding several strings to `ur.json`). Then, there should be a translation banner, but the strings will be mostly in English.
Should we open a separate issue for that?